crates/astria-core/src/bridge.rs

```diff
@@ -12,3 +14,18 @@ pub struct Ics20WithdrawalFromRollupMemo {
     pub block_number: u64,
     pub transaction_hash: [u8; 32],
 }
+
+/// Memo format for a ICS20 transfer to Astria which is sent to a
+/// bridge account, which will then be deposited into the rollup.
+#[derive(Debug, Clone)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize),
+    derive(serde::Deserialize)
+)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
```

Comment:
I got hung up on this, but it turns out that `camelCase` is the actually correct pbjson mapping, whereas we use `preserve_proto_field_names` in the protobuf generated. I don't recall why I did that, but I shouldn't have done that.

Also a note on `deny_unknown_fields`: this makes schema evolution extremely tricky and will require a breaking change on every change to this memo. Not that it's wrong - just want to flag it.